temp_library/scene_collection.py

```diff
@@ -0,0 +1,81 @@
+from temp_library.scene import *
+from temp_library.option import *
+from temp_library.decision import *
+from temp_library.helpers import openscrn
+from temp_library.chance_option import ChanceOptions
+
+
+class FleeScene(Scene):

```

Comment:
Put this file in an example directory + including all other files that are examples. No need to interfere with the main source code of the library.